components/app-layouts/app.tsx

```diff
@@ -88,6 +88,7 @@ export const AppLayout: ComponentWithChildren<AppLayoutProps> = (props) => {
       new QueryClient({
         defaultOptions: {
           queries: {
+            refetchOnWindowFocus: false,
```

Comment:
this means *none* of our queries will ever re-fetch when window re-focusses. that might be OK but affects more than just the one spot so let's not take this decision without considering it first (not sure if you have already)

Reply:
No longer need this - found the proper way to fix the issue. Closing this